src/auth/Loading.tsx

```diff
@@ -0,0 +1,7 @@
+export function Loading() {
+  return (
+    <>
+      <h3>LOADING .....</h3>
```

Comment:
You can use loader here

Reply:
Just noticed that this file wasn't being used.
Thanks for spotting it.